client/cli/src/runner.rs

```diff
@@ -142,9 +142,9 @@ impl<C: SubstrateCli> Runner<C>
 			C::copyright_start_year(),
 			Local::today().year()
 		);
-		info!("Chain specification: {}", self.config.chain_spec.name());
-		info!("Node name: {}", self.config.network.node_name);
-		info!("Roles: {}", self.config.display_role());
+		info!("📋 Chain specification: {}", self.config.chain_spec.name());
+		info!("🏷 Node name: {}", self.config.network.node_name);
+		info!("👤 Roles: {}", self.config.display_role());
```

Comment:
I like this. Very cute!

Reply:
huhu that's not me, it was on master. I just updated the code consistently with what is on master